components/03-modules/dashboard-form/_dashboard-form.scss

```diff
@@ -0,0 +1,37 @@
+$dashborad-form-margin: 30px !default;
+
+.dashboard-form {
+    &--hidden {
+        display: none;
+    }
+
+    &__title {
+        margin-bottom: $dashborad-form-margin;
+        padding-bottom: 10px;
+        border-bottom: 1px solid $gray;
+    }
+
+    &__input {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__fields {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__checkbox {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__content {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    &__title-form {
+        margin-bottom: $dashborad-form-margin;
+    }
+
+    .select {
+        margin-bottom: $dashborad-form-margin;
+    }
```

Comment:
if you added the same margin for each element, couldn't you limit the class name and add, for example only one class to the fields? instead of making separate class for input, select and checkbox field?

Reply:
Yes It's true, I can save some lines of code. Now it's one class which calls `divider`

Reply:
and changed in other files too

Reply:
You can limit the code by connecting `&__divider` and `.select` like:
```
&__divider,
.select {
    margin-bottom: $dashborad-form-margin;
}
```
IMO instead of `.select` we should use additional class (like `dashboard-form__select` or `dashboard-form__field` or if it's only for padding, maybe it will be sufficient to add to select `dashboard-fomr__divider`. What do you think?